src/backend/utils/html/modify-pre.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Take a <pre>...</pre> and return <pre><code>...</code></pre>
+ */
+function insertCode(pre, document) {
+  const codeElem = document.createElement('code');
+  codeElem.innerHTML = pre.innerHTML;
+  pre.innerHTML = codeElem.outerHTML;
+}
+/**
+ * Given a parsed JSDOM Object. find all <pre></pre> tags without child code elements and modify the
+ * Pre tag to have inner <code></code> tags which wraps around the previous content
+ * resulting in <pre><code>...</code></pre>
+ */
+module.exports = function (dom) {
+  if (!(dom && dom.window && dom.window.document)) {
+    return;
+  }
+
+  dom.window.document.querySelectorAll('pre').forEach((pre) => {
+    // no child elements
+    if (pre.children.length <= 0) insertCode(pre, dom.window.document);
+    // has child <br> elements
+    else if (pre.children[0].nodeName === 'BR') {
+      pre.innerHTML = pre.innerHTML.replace(/<br>/g, '\n'); // trim
```

Comment:
The one downside I can see to this is that it will be impossible to have a `<br>` in an HTML code block.  We have no way to tell a literal `<br>` from one used to add a new line.

Reply:
That's a good point, the best choice probably is to delete the line and just live with the new line `<br>` tags in code.. atleast for now..

Reply:
Only 1 post from 2013 uses `<br>` as a literal as pointed out by @humphd on Slack. If there happens to be unwanted side effects within the posts in the future because of this fix, we will address them in another issue.